api/report_server.thrift

```diff
@@ -298,6 +305,26 @@ service codeCheckerDBAccess {
                                              4: ReportFilterList reportFilters)
                                              throws (1: shared.RequestFailed requestError),
 
+  Counters getSeverityCounts(1: list<i64> runIds,
```

Comment:
The conversion is always the clients resposibility as before. I do not wanted to create a separate counter type with both values as int.